src/dsql/DdlNodes.h

```diff
@@ -234,7 +233,18 @@ class RecreateNode : public DdlNode
 protected:
 	virtual void putErrorPrefix(Firebird::Arg::StatusVector& statusVector)
 	{
-		statusVector << Firebird::Arg::Gds(ERROR_CODE) << createNode->name;
+		statusVector << Firebird::Arg::Gds(ERROR_CODE) << nameToString(createNode->name);
+	}
+
+private:
+	Firebird::string nameToString(const QualifiedName& name)
+	{
+		return name.toQuotedString();
+	}
+
+	Firebird::string nameToString(const MetaName& name)
+	{
+		return name.toQuotedString();
```

Comment:
Are these methods really needed, provided that they're one-liners that could be used directly instead?

Reply:
No. It was necessary in a previous version where the `toQuotedString` methods had different names in the two classes.